source/getting-started/use_cases/cloud_security.rst

```diff
@@ -100,20 +100,20 @@ Example of AWS dashboard:
    :align: center
    :width: 100%
 
-More information about how does Wazuh monitor AWS can be found at our :ref:`documentation <amazon>`.
+More information on how Wazuh monitors AWS can be found at our :ref:`documentation <amazon>`.
 
 Microsoft Azure
 ---------------
 
-The :ref:`Wazuh agent <wazuh_agent>` module for Microsoft Azure makes it easy to pull Azure platform logs. More specifically, it is capable of pulling data from the following services:
+The :ref:`Wazuh agent <wazuh_agent>` module for Microsoft Azure makes it easy to pull Azure platform logs.  In particular, it is designed to obtain data from the following services:
 
 - :ref:`Log Analytics API <azure_monitoring_activity>`: The Log Analytics API is a core component of the Azure Monitor service, which is used to aggregate and analyze log data. The sources of such data are cloud applications, operating systems and Azure resources. The Wazuh module for Azure is capable of querying the Log Analytics API, pulling the logs that are collected by the Azure monitor service.
 
-- :ref:`Blob Storage API <azure_monitoring_activity>`: Logs from Azure services can optionally be pushed to Azure Blob Storage. More specifically, you can configure an Azure service to export logs to a container in a storage account created for that purpose. Then, the Wazuh agent can download those logs via its integration with the Blob Storage API.
+- :ref:`Blob Storage API <azure_monitoring_activity>`: Logs from Azure services are optionally pushed to Azure Blob Storage. Specifically, it is possible to configure an Azure service to export logs to a container in a storage account created for that purpose. Afterwards, the Wazuh agent will download those logs via its integration with the Blob Storage API.
```

Comment:
This is not compulsory, so it should remain as it was previously.

Reply:
The expression has not changed in the new version. It says "it is possible". It's only a rephrasing of the way to express it, but it's not changing the "optional" character of the sentence. It says "are optionally pushed".